platform/ui/src/assets/icons/tool-circle.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32">
```

Comment:
where are these 4 icons being used?

Reply:
They are used by annotation tools for dicom microscopy.
Removing unused icons in "mode"